pkg/resources/scim_integration_acceptance_test.go

```diff
@@ -1,79 +1,180 @@
+	t.Cleanup(roleCleanup)
+	acc.TestClient().Role.GrantRoleToCurrentRole(t, role.ID())
+	id := acc.TestClient().Ids.RandomAccountObjectIdentifier()
+	m := func() map[string]config.Variable {
+		return map[string]config.Variable{
+			"name":                config.StringVariable(id.Name()),
+			"enabled":             config.BoolVariable(false),
+			"scim_client":         config.StringVariable(string(sdk.ScimSecurityIntegrationScimClientGeneric)),
+			"sync_password":       config.BoolVariable(false),
+			"network_policy_name": config.StringVariable(networkPolicy.Name),
+			"run_as_role":         config.StringVariable(role.ID().Name()),
+			"comment":             config.StringVariable("foo"),
+		}
 	}
+	resource.Test(t, resource.TestCase{
+		ProtoV6ProviderFactories: acc.TestAccProtoV6ProviderFactories,
+		PreCheck:                 func() { acc.TestAccPreCheck(t) },
+		TerraformVersionChecks: []tfversion.TerraformVersionCheck{
+			tfversion.RequireAbove(tfversion.Version1_5_0),
+		},
+		Steps: []resource.TestStep{
+			{
+				ConfigDirectory: acc.ConfigurationDirectory("TestAcc_ScimIntegration/complete"),
+				ConfigVariables: m(),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("snowflake_scim_integration.test", "name", id.Name()),
+					resource.TestCheckResourceAttr("snowflake_scim_integration.test", "enabled", "false"),
+					resource.TestCheckResourceAttr("snowflake_scim_integration.test", "scim_client", "GENERIC"),
+					resource.TestCheckResourceAttr("snowflake_scim_integration.test", "run_as_role", role.Name),
+					resource.TestCheckResourceAttr("snowflake_scim_integration.test", "network_policy", networkPolicy.Name),
+					resource.TestCheckResourceAttr("snowflake_scim_integration.test", "sync_password", "false"),
+					resource.TestCheckResourceAttr("snowflake_scim_integration.test", "comment", "foo"),
+					resource.TestCheckResourceAttrSet("snowflake_scim_integration.test", "created_on"),
+				),
+			},
+			{
+				ConfigDirectory:   acc.ConfigurationDirectory("TestAcc_ScimIntegration/complete"),
+				ConfigVariables:   m(),
+				ResourceName:      "snowflake_scim_integration.test",
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
 
-	resource "snowflake_scim_integration" "test" {
-		name = "%s"
-		scim_client = "AZURE"
-		provisioner_role = snowflake_role.azure.name
-		network_policy = snowflake_network_policy.azure.name
-		depends_on = [
-			snowflake_grant_privileges_to_account_role.azure_grants,
-			snowflake_grant_account_role.azure
-		]
+func TestAcc_ScimIntegration_invalid(t *testing.T) {
+	id := acc.TestClient().Ids.RandomAccountObjectIdentifier()
+	m := func() map[string]config.Variable {
+		return map[string]config.Variable{
+			"name":                config.StringVariable(id.Name()),
+			"enabled":             config.BoolVariable(false),
```

Comment:
let's not put enabled (another error should pop out)

Reply:
Done in a separate test case because enum validations errors don't appear together with it.